docs/user-guide/docker-plugin-drivers.md

```diff
@@ -43,3 +43,5 @@ Note:
 1. The current version of the Photon volume plugin doesn't support authentication as yet, this will be added up in a subsequent release.
 2. The "target port" is set to "9000" if authentication isn't enabled in photon controller, else port "443" is used.
 3. Both project and VM IDs mentioned above can be obtained via the "photon" CLI, see https://github.com/vmware/photon-controller/wiki/Compile-Photon-Controller-CLI.
+4. The plugin driver supports both "vsphere" and "vmdk" as driver names and both can be used interchangeably for now. The "vmdk" driver name will be deprecated in a later release, allowing users to use only "vsphere" as the driver name.
+5. The "vmdk" driver is however the default driver name that is used when the plugin is installed.
```

Comment:
I did not find the explanation of how to switch between vmdk and vsphere (i.e `edit xxx file; restart the service` or something along these lines